### DISCUSSION TITLE
2024/07/01/CVE-2024-4577%E5%A4%8D%E7%8E%B0/

### DISCUSSION THREAD

**Discussion by giscus** [argvchs/giscus-comments#48] [Announcements]
# 2024/07/01/CVE-2024-4577%E5%A4%8D%E7%8E%B0/

这个网站没有描述

https://xqhff.github.io/2024/07/01/CVE-2024-4577%E5%A4%8D%E7%8E%B0/

**Comment by Hi-ANINEG** [comment]
太酷了
